main.tf

```diff
@@ -140,12 +140,13 @@ module "server" {
   server_private           = var.server_private
   cloudsql_proxy_arch      = var.cloudsql_proxy_arch
   cloud_profiler_enabled   = var.cloud_profiler_enabled
+  mailjet_enabled          = var.mailjet_enabled
```

Comment:
Can we also update the `README.md` file?

Like the example here?
https://github.com/forseti-security/terraform-google-forseti/pull/297/files

Reply:
@jf-marquis-Adeo, just FYI, the readme is updated by running `make generate_docs` in the root of the module.